### ISSUE TITLE
systemctl edit creates SUID files

### ISSUE THREAD

**Issue by AladW** [systemd/systemd#2538]
Running 

```
sudo systemctl edit connman
```

with the following contents:

```
[Service]
ExecStart=
ExecStart=/usr/bin/connmand -n --nodnsproxy
```

resulted in a suid .conf file being created in /etc/systemd/system/connman.service.d

```
  File: ‘override.conf’
  Size: 65          Blocks: 8          IO Block: 4096   regular file
Device: 802h/2050d  Inode: 6824474     Links: 1
Access: (7755/-rwsr-sr-t)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2016-02-05 22:56:31.169881058 +0100
Modify: 2016-02-05 22:56:30.389881043 +0100
Change: 2016-02-05 22:56:31.129881058 +0100
 Birth: -
```

Using systemd 228 on Arch Linux:

```
/usr/lib/systemd/systemd --version
systemd 228
+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN
```

**Comment by ohsix** [comment]
try SYSTEMD_EDITOR=umask systemctl edit connman

it will print the current default file creation mode

i did a quick grep and nothing sets the high bit in systemd that i could see, it could come from your shell / .profile (run umask there to see?)

**Comment by AladW** [comment]
Running `SYSTEMD_EDITOR=umask systemctl edit connman` in a root shell opens the override file in nano for me, as without specifying SYSTEMD_EDITOR.

`umask` gives 0022

My profile file:
https://projects.archlinux.org/svntogit/packages.git/tree/trunk/profile?h=packages/filesystem

**Comment by poettering** [comment]
Pretty sure this already got fixed by 06eeacb6fe029804f296b065b3ce91e796e1cd0e
